recipes/python-periodictable/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-periodictable" %}
```

Comment:
does that go for the name of the folder as well? (I suspect that there may be other packages for other languages that share this name)